adafruit_bmp280.py

```diff
@@ -433,15 +446,15 @@ class Adafruit_BMP280_I2C(Adafruit_BMP280):  # pylint: disable=invalid-name
 
     """
 
-    def __init__(self, i2c, address=0x77):
+    def __init__(self, i2c: int, address=0x77):
```

Comment:
`i2c` should be of type `busio.I2C`, so you can import it up top with `from busio import I2C` and then do:

```suggestion
    def __init__(self, i2c: I2C, address: int = 0x77) -> None:
```